metisp/pymetis/src/pymetis/recipes/lm_img/metis_lm_img_basic_reduce.py

```diff
@@ -141,19 +117,52 @@ def process_images(self) -> [PipelineProduct]:
         a single combined frame that is used throughout the pipeline.
         """
 
-        Msg.info(self.__class__.__qualname__, f"Starting processing image attribute.")
+        Msg.info(self.__class__.__qualname__, f"Processing Images")
+
+        Msg.info(self.__class__.__qualname__, f"Loading calibration files")
 
         flat = cpl.core.Image.load(self.inputset.master_flat.frame.file, extension=0)
         bias = cpl.core.Image.load(self.inputset.master_dark.frame.file, extension=0)
         gain = cpl.core.Image.load(self.inputset.gain_map.frame.file, extension=0)
 
         Msg.info(self.__class__.__qualname__, f"Detector name = {self.detector}")
-
-        flat = self.prepare_flat(flat, bias)
+        
+        Msg.info(self.__class__.__qualname__, f"Loading raw images")
         images = self.prepare_images(self.inputset.raw.frameset, flat, bias)
+        Msg.info(self.__class__.__qualname__, f"Pretending to correct crosstalk")
+        Msg.info(self.__class__.__qualname__, f"Pretending to correct for linearity")
+
+        Msg.info(self.__class__.__qualname__, f"Subtracting Dark")
+
+        images.subtract_image(bias)
+
+        Msg.info(self.__class__.__qualname__, f"Flat fielding")
+
+        images.divide_image(flat)
+
+        Msg.info(self.__class__.__qualname__, f"Pretending to remove masked regions")
+
+        Msg.info(self.__class__.__qualname__, f"Combining Images")
         combined_image = self.combine_images(images, self.parameters["metis_lm_img_basic_reduce.stacking.method"].value)
+        
         header = cpl.core.PropertyList.load(self.inputset.raw.frameset[0].file, 0)
 
+        Msg.info(self.__class__.__qualname__, f"Pretending to calculate noise")
+        
+        Msg.info(self.__class__.__qualname__, f"Pretending to calculate bad pixels")
+        
+        Msg.info(self.__class__.__qualname__, f"Actually Calculating QC Parameters")
+
+        qc_lm_img_median = combined_image.get_median()
+        qc_lm_img_std = combined_image.get_stdev()
+        qc_lm_img_max = combined_image.get_max()
+
+        Msg.info(self.__class__.__qualname__, f"Appending QC Parameters to header")
+
+        header.append("QC_LM_IMG_MEDIAN",qc_lm_img_median)
```

Comment:
This works, but it might be better to append a `cpl.Property` structure, in particular because it allows adding a comment, which is probably useful to the end user:
```
header.append(cpl.core.Property("QC LM IMG MEDIAN",    # no underscores; is "LM IMG" necessary?
                           cpl.core.Type.DOUBLE,
                           combined_image.get_median(),    # actually individual image
                           "[ADU] median value of image")
```